src/metabase/query_processor/middleware/parameters/sql.clj

```diff
@@ -228,6 +228,45 @@
         ;; otherwise just return the single number
         (first parts)))))
 
+(defn numericString?
+  "Return true if s is a numeric string"
+  [s]
+  (and (string? s)
+       (if-let [s (seq s)]
```

Comment:
Use `when` instead of `if` (and `when-let` instead of `if-let`) if there's only a single arg in the body